core/modules/filter/filter.admin.inc

```diff
@@ -474,6 +503,20 @@ function filter_admin_format_form_submit($form, &$form_state) {
   $form_state['redirect'] = 'admin/config/content/formats';
 }
 
+/**
+ * Page callback: Enable a text format.
+ */
+function filter_admin_enable($format = NULL) {
+  if (isset($format)) {
```

Comment:
We need to add token validation on these links to prevent CSRF attacks. We just did this over in Flexible Layouts in this commit: https://github.com/backdrop/backdrop/pull/2646/commits/45ab3a029a9f593fdeea024ba2a1df420a7d0d43#diff-1b45663e4cee17ec13f8e83d295a118cR740

Reply:
Looks good!